### PR TITLE
Remove leading spaces

### PR DESCRIPTION
**Pull request by grumdrig**
Fixed 8 erroneous entries which had leading spaces. 

Mostly they were duplicates of the correctly spelled entries, except " victimised" which didn't have a correctly spelled entry. Many had more complete listings; the information was moved to its correct place. I kept the better definitions and usage examples in each case. 

I kept the definition of "very important person" which didn't include "(and often overbearing)" which seemed extraneous to the defintion of the word. 

Also fixed grammar in the definitions of analog/analogue, and corrected typos. And fixed errors in synonyms which referred to the erroneous entries, etc.

### CODE REVIEW

File: data/a.json [hunk 1]
```diff
@@ -11271,11 +11271,17 @@
...
+                "name": "American",
+                "is_dialect": true
+            }
+        ],
         "meanings": [
             {
                 "id": "bcee189e72",
                 "def": "of a circuit or device having an output that is proportional to the input",
-                "example": "analogue device",
+                "example": "This analog device picks up radio signals from all over the world.",
```
Comment (EricNelson12) [inline data/a.json:11284]: Nice!

File: data/a.json [hunk 2]
```diff
@@ -11284,6 +11290,7 @@
             {
                 "id": "cc1a78ef04",
                 "def": "something having the property of being analogous to something else",
+                "example": "That argument is an analog of the one we just proved wrong.",
```
Comment (EricNelson12) [inline data/a.json:11293]: https://grammarist.com/spelling/analog-analogue/#:~:text=The%20spellings%20are%20largely%20interchangeable,all%20senses%20of%20the%20word.

```suggestion
                "example": "That argument is an analogue of the one we just proved wrong.",
```

File: data/v.json [hunk 3]
```diff
@@ -9612,13 +9644,17 @@
...
+        "editors": [
+            "lefurjah"
+        ],
+        "contributors": [
+            "luxfactaest",
+            "msingle",
+            "zellerpress"
         ]
     },
     "vaporisation": {
```
Comment (EricNelson12) [inline data/v.json:9660]: Is this even a word?

File: data/v.json [hunk 4]
```diff
@@ -30789,6 +30830,21 @@
                 "synonyms": [
                     "masculinization"
                 ]
+            },
+            {
+                "id": "37b9739b1f",
+                "def": "the normal development of male sexual characteristics in a male",
```
Comment (EricNelson12) [inline data/v.json:30836]: This definition seems wrong from my research.

```suggestion
                "def": "the development of masculine characteristics in a female",
```

**Review by EricNelson12 — COMMENTED** [review]
Nice changes! Left a few comments

File: data/v.json (hunk 3 above)
Comment (grumdrig) [reply data/v.json:9660]: vaporization/vaporisation? I believe so...

File: data/a.json (hunk 2 above)
Comment (grumdrig) [reply data/a.json:11293]: This example is within the definition of the word that uses that spelling. That is to say, it doesn't make sense to have an example with "analog" withing the definition of "analogue"

File: data/v.json (hunk 4 above)
Comment (grumdrig) [reply data/v.json:30836]: (I'm not good with github. Tell me if I'm supposed to do something to accept this change.)

Comment (EricNelson12) [reply data/v.json:30836]: There should be a button with "Accept suggested change" or you can just make the change and push it up. I just realized that 

>Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

And that's not me, lol, so I'm not sure if we can even push / contribute. 😢

File: data/v.json (hunk 3 above)
Comment (HamptonMakes) [reply data/v.json:9660]: British version!

**Review by HamptonMakes — APPROVED** [review]

**Pull request merged**